config/application.rb

```diff
@@ -76,10 +76,10 @@ class Application < Rails::Application
       Spree::Config['checkout_zone'] = ENV['CHECKOUT_ZONE']
       Spree::Config['currency'] = ENV['CURRENCY']
 
-      if ActiveRecord::Base.connected? && Spree::Country.table_exists?
+      begin
         country = Spree::Country.find_by(iso: ENV['DEFAULT_COUNTRY_CODE'])
         Spree::Config['default_country_id'] = country.id if country.present?
```

Comment:
Now that we've wrapped all this default country stuff in that nice service, do we even need to set this here? Or can we cut this whole begin/rescue block completely and not set `Spree::Config['default_country_id']` at all?

Reply:
:fire::tada: